guides/05_tabular_data_science_and_plots/using_gradio_with_bigquery.md

```diff
@@ -0,0 +1,110 @@
+# Using Gradio with Google BigQuery
+
+Google BigQuery is a cloud-based big data analytics web service for processing very large data sets. It is a serverless and highly scalable data warehousing solution that enables users to analyze data using SQL-like queries.
+
+In this tutorial, we will show you how to query a BigQuery dataset in Python and display the data *real-time* in a  dashboard using `gradio`. 
```

Comment:
Let's try a summary of key points in the blog here. Something like: 

> How to use Google BigQuery: 
> 1. Setting up Google BigQuery Credentials 
> 2. Using the BigQuery Client 
> 3. Building a Dashboard

Reply:
Let's do it